release/artifacts.yaml

```diff
@@ -33,15 +33,19 @@
   type: jar-scala
 - target: //daml-lf/data-scalacheck:data-scalacheck
   type: jar-scala
+- target: //daml-lf/encoder:encoder
+  type: jar-scala
 - target: //daml-lf/engine:engine
   type: jar-scala
 - target: //daml-lf/interface:interface
   type: jar-scala
 - target: //daml-lf/interpreter:interpreter
   type: jar-scala
+- target: //daml-lf/kv-transaction-support:kv-transaction-support
+  type: jar-scala
 - target: //daml-lf/language:language
   type: jar-scala
-- target: //daml-lf/kv-transaction-support:kv-transaction-support
+- target: //daml-lf/parser:parser
```

Comment:
Same question here, why do we need this? This shouldn’t be something you rely on outside of `//daml-lf`.